js/index.js

```diff
@@ -0,0 +1,310 @@
+const baseUrl = 'https://trektravel.herokuapp.com/trips';
```

Comment:
since this is a constant whose value is a string, it might be nice to return to the naming convention `BASE_URL`